Kickstarter-iOS/Views/Controllers/ChangeEmailViewController.swift

```diff
@@ -112,8 +113,12 @@ internal final class ChangeEmailViewController: UIViewController {
 
     _ = self.passwordTextField
       |> passwordFieldStyle
```

Comment:
Since the password fields on Settings have similar properties, we could create a property on `SettingsStyles.swift` like:
```
public let settingsPasswordFieldStyle = passwordFieldStyle
  <> settingsFormFieldStyle
  <> UITextField.lens.textAlignment .~ .right
```
And simply use:
```
_ = self.passwordTextField
  |>  settingsPasswordFieldStyle
```
And just set new properties as needed, such as `returnKeyType`.